query_server/sqllogicaltests/cases/stream/stream_table_create.slt

```diff
@@ -0,0 +1,37 @@
+statement ok
+DROP TABLE IF EXISTS t1;
+
+statement ok
+CREATE TABLE t1
+(pressure DOUBLE, temperature DOUBLE, visibility DOUBLE, TAGS('station'));
+
+
+query T
+DESCRIBE TABLE t1;
+----
+time TIMESTAMP(NANOSECOND) TIME DEFAULT
+station STRING TAG DEFAULT
+pressure DOUBLE FIELD DEFAULT
+temperature DOUBLE FIELD DEFAULT
+visibility DOUBLE FIELD DEFAULT
+
+
+statement ok
+DROP TABLE IF EXISTS t1_stream;
+
+statement error Field or Tag have the same name
+CREATE STREAM TABLE t1_stream(time TIMESTAMP,time TIMESTAMP, station STRING, pressure DOUBLE, temperature DOUBLE, visibility DOUBLE) 
+    WITH (db = 'public', table = 't1', event_time_column = 'time')
+    engine = tskv;
+
+
+statement ok
+DROP TABLE IF EXISTS t1_stream;
+
+
+statement error Invalid option
+CREATE STREAM TABLE t1_stream(time TIMESTAMP, station STRING, pressure DOUBLE, temperature DOUBLE, visibility DOUBLE) 
+    WITH (db = 'public', table = 't1', event_time_column = 'pressure')
+    engine = tskv;
+
```

Comment:
why this sql executed failed?

Reply:
because the type of the 'event_time_column' should be  timestamp and the 'pressure' is double